.github/workflows/deploy.yml

```
@@ -0,0 +1,56 @@
+          key: ${{ secrets.SSH_KEY }}
+          port: 22
+          script_stop: true
+          script: |
+            sudo yum update -y && sudo yum install -y java-21-amazon-corretto
+            for pid in $(pgrep java); do
+              if ps -p $pid -o args= | grep -q 'java -jar'; then
+                echo "Java process with 'java -jar' found (PID: $pid). Terminating..."
+                kill -9 $pid
+              fi
+            done
+            echo "nohup java -jar ~/*.jar > ~/app.log 2>&1 &" | at now
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)